src/components/PanelSection.tsx

```diff
@@ -23,14 +26,29 @@ export function PanelSection({
     >
       <div
         className={cx(
+          'group/header',
```

Comment:
Is this from tailwind? Because you haven't defined any rule for the class

Reply:
Yes, I also learned this one today :-). We need this bc there are two nested groups.